linux-anvil-cuda/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN rpm -e --nodeps --verbose gcc gcc-c++
 RUN yum update -y && \
     yum install -y \
         bzip2 \
+        findutils \
```

Comment:
Please use the conda-forge package: https://anaconda.org/conda-forge/findutils

Reply:
I'd rather not, for now.
- `conda install findutils` would only add unneeded baggage.
- `yum install findutils` is a no-op current since it is already installed.
- It is in the same league as `bzip2`, `tar` => we only install those to run the Miniconda install, but can (or rather should, IMO) ditch them if/when we use multi-stage builds (where we needn't run the installer at all).

So, if you feel strongly about not running `yum install findutils`, then I'd rather say we neither run `yum install findutils` nor `conda install findutils`: The images will still build fine since `find` is already part of the base image, we would just be a little less verbose/explicit ( https://github.com/conda-forge/docker-images/pull/107/files#r299215307 ).